src/YieldDistributor.sol

```diff
@@ -121,6 +131,14 @@ contract YieldDistributor is OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
+        /*
+        - This returns the voting power based on the last period, which is intended but may be
+          surprising. It would be good to comment this behavior.
```

Comment:
Voting power at the time of distribution required us to iterate over all the voters, introducing an attack vector. This prompted us to record voting power at the time of voting and not distribution. This would, however, introduce an advantage to "late voters", so we opted to calculate it at a set delay.

Reply:
Makes sense, just think it deserves a comment.